### ISSUE TITLE
`/p clear` not unmerging plots

### ISSUE THREAD

**Issue by Xaver106** [IntellectualSites/PlotSquared#3733]
### What feature do you want to see added?

`/p clear` also unmerges plots. This can sometimes be useful, but most of the time it is unnecessary and annoying. Having `/p clear` not unmerge the plots would be a big improvement.

### Are there any alternatives?

not really, maybe a custom plugin using the PS API.

**Comment by ch4ika** [comment]
Yes, this feature would also be useful for me, since we have many players who always build maps and it is impractical that the merges always disappear, since these users then have to reclaim plots again and again on their own.

**Comment by NotMyFault** [comment]
Instead of breaking existing and known behavior, this could be implemented as optional flag, like `/plot clear unlink`.

**Comment by RedstoneFuture** [comment]
> Instead of breaking existing and known behavior, this could be implemented as optional flag, like `/plot clear unlink`.

I know what do you mean. But the current behavior seems illogical in my opinion for the normal clear command. When I want to clear the plot, I only want to remove the contents (blocks and entities) of the plot (= `/p clear`). If I want to unlink it, then I want to undo the merge (= `/p unlink`) - maybe with a clear before (= e.g. `/p unlink clear`).

**Comment by dordsor21** [comment]
I think a config option to change the default for whether a merge is removed on clear or not would be a good idea. Also implementing a `/plot unlink <clear>` option/command would be a good implementation (also possibly configurable of the default, otherwise the player must give a true|false)
